main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as read_file:
+        data = json.load(read_file)
+        for nickname, detail_info in data.items():
+            race = detail_info.get("race")
+            new_race, created = Race.objects.get_or_create(
+                name=race.get("name"),
+                description=race.get("description")
+            )
+
+            skills = race.get("skills")
+            for skill in skills:
+                Skill.objects.get_or_create(
+                    race=new_race,
+                    name=skill.get("name"),
+                    bonus=skill.get("bonus")
+                )
+
+            guild = detail_info.get("guild")
+            if guild is not None:
+                new_guild, created = Guild.objects.get_or_create(
+                    name=guild.get("name"),
+                    description=guild.get("description")
+                )
+            else:
```

Comment:
this else is useless, because you can use guild

Reply:
but when I delete else, i have 1 mistake like this:  
assert [('john', 'jo...blacksmiths')] == [('john', 'jo...human', None)]
E         At index 4 diff: ('nick', 'nick@gmail.com', "Hello, I'm Nick", 'human', 'blacksmiths') != ('nick', 'nick@gmail.com', "Hello, I'm Nick", 'human', None)
E         Use -v to get more diff

Reply:
done